web-components/src/components/icons/CreditClassIcon.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import SvgIcon from '@mui/material/SvgIcon';
+import { Theme } from '~/theme/muiTheme';
+import { SxProps } from '@mui/material';
+
+interface IconProps {
+  sx?: SxProps<Theme>;
+}
+
+export function CreditClassIcon({ sx = [] }: IconProps): JSX.Element {
```

Comment:
You can just pass `SvgIconProps`(imported from MUI) directly to this component and simplify imports, FYI, ie
```tsx
export function CreditClassIcon(props: SvgIconProps): JSX.Element {
return (
    <SvgIcon
      {...props}
      viewBox="0 0 20 17"
```

Reply:
i like it